packages/web/src/store/tipping/sagas.ts

```diff
@@ -223,6 +230,9 @@ function* fetchSupportingForUserAsync({
     return
   }
 
+  // todo: need to also get whether logged in user is supporting this user
```

Comment:
this is what we talked about re fetching support data given sender and receiver. i will follow up